.github/actions/gh_create_issue/create_issue.sh

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+set -x
```

Comment:
I think that the `-x` output is a bit much in the CI. Should we drop the `-x`. We are already printing warning and debug messages anyway.